system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_kusama_config.rs

```diff
@@ -0,0 +1,400 @@
+	},
+};
+use cumulus_primitives_core::ParentThen;
+use frame_support::{parameter_types, traits::PalletInfoAccess};
+use polkadot_runtime_constants as constants;
+use sp_runtime::{traits::ConstU32, RuntimeDebug};
+use xcm::{
+	latest::prelude::*,
+	prelude::{InteriorMultiLocation, NetworkId},
+};
+use xcm_builder::{BridgeBlobDispatcher, HaulBlobExporter};
+
+/// Lane identifier, used to connect Polkadot Asset Hub and Kusama Asset Hub.
+pub const XCM_LANE_FOR_ASSET_HUB_POLKADOT_TO_ASSET_HUB_KUSAMA: LaneId = LaneId([0, 0, 0, 1]);
+
+// Parameters that may be changed by the governance.
+parameter_types! {
+	/// Reward that is paid (by the Polkadot Asset Hub) to relayers for delivering a single
+	/// Polkadot -> Kusama bridge message.
+	///
+	/// This payment is tracked by the `pallet_bridge_relayers` pallet at the Polkadot
+	/// Bridge Hub.
+	pub storage DeliveryRewardInBalance: Balance = constants::currency::UNITS / 2_000;
+
+	/// Registered relayer stake.
+	///
+	/// Any relayer may reserve this amount on his account and get a priority boost for his
+	/// message delivery transactions. In exchange, he risks losing his stake if he would
+	/// submit an invalid transaction. The set of such (registered) relayers is tracked
+	/// by the `pallet_bridge_relayers` pallet at the Polkadot Bridge Hub.
+	pub storage RequiredStakeForStakeAndSlash: Balance = 500 * constants::currency::UNITS;
+}
+
+// Parameters, used by both XCM and bridge code.
+parameter_types! {
+	/// Kusama Network identifier.
+	pub KusamaGlobalConsensusNetwork: NetworkId = NetworkId::Kusama;
+	/// Interior location (relative to this runtime) of the with-Kusama messages pallet.
+	pub BridgePolkadotToKusamaMessagesPalletInstance: InteriorMultiLocation = X1(
+		PalletInstance(<BridgeKusamaMessages as PalletInfoAccess>::index() as u8),
+	);
+
+	/// Identifier of the sibling Polkadot Asset Hub parachain.
+	pub AssetHubPolkadotParaId: cumulus_primitives_core::ParaId = polkadot_runtime_constants::system_parachain::ASSET_HUB_ID.into();
+	/// A route (XCM location and bridge lane) that the Polkadot Asset Hub -> Kusama Asset Hub
+	/// message is following.
+	pub FromAssetHubPolkadotToAssetHubKusamaRoute: SenderAndLane = SenderAndLane::new(
+		ParentThen(X1(Parachain(AssetHubPolkadotParaId::get().into()))).into(),
+		XCM_LANE_FOR_ASSET_HUB_POLKADOT_TO_ASSET_HUB_KUSAMA,
+	);
+}
+
+// Parameters, used by bridge transport code.
+parameter_types! {
+	/// Number of Kusama headers to keep in the runtime storage.
+	///
+	/// Note that we are keeping only required header information, not the whole header itself. Roughly, it
+	/// is the 2 hours of real time (assuming that every header is submitted).
+	pub const RelayChainHeadersToKeep: u32 = 1_200;
+	/// Number of Kusama Bridge Hub headers to keep in the runtime storage.
+	///
+	/// Note that we are keeping only required header information, not the whole header itself. Roughly, it
+	/// is the 2 hours of real time (assuming that every header is submitted).
```

Comment:
But 1 hour are 600 blocks, so shouldn't this be 1200 for keeping 2 hours?

Reply:
Ahh, `RelayChainHeadersToKeep` is correct. I assume this still assumes that there are only a parachain block every 12 seconds? If yes, this is not correct anymore and we should update this as well to 1200.